src/app/story/directives/podcast.component.html

```diff
@@ -1,5 +1,10 @@
 <publish-spinner *ngIf="!episode"></publish-spinner>
 <form *ngIf="episode">
+  <publish-fancy-field label="Episode URL" textinput [model]="episode" name="webLink">
```

Comment:
I don't think this should be at the top. Seems like it will rarely be used. #338 doesn't exactly specify that explicit should be at the top, but it seems implied that it's the most often used field and should be at the top (above author info.) I don't imagine episode URL will be set frequently.